goentri.com.clickfunnels.json

```diff
@@ -8,13 +8,14 @@
     "description": "Allows user to easily set up domain using Entri",
     "syncPubKeyDomain": "goentri.com",
     "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com",
-    "variableDescription": "verificationValue is the value that is used to verify domain ownership,clusterValue is the cluster assigned to the domain,websiteId is the id of website, keyValue is the generated key value",
+    "variableDescription": "verificationValue is the value that is used to verify domain ownership,clusterValue is the cluster assigned to the domain,websiteId is the id of website, keyValue is the generated key value, hosth is the website host value",
     "sharedProviderName": true,
     "hostRequired":true,
+    "multiInstance": true,
     "records": [
       {
         "type": "TXT",
-        "host": "%domain%",
+        "host": "%hosth%",
```

Comment:
Not really the intended change.
See Slack conversation for details https://domainconnect.slack.com/archives/C0399E15DQC/p1650581557086779?thread_ts=1650552627.610689&cid=C0399E15DQC